station/Classes/Presentation/Modules/TagSettings/Submodules/UpdateFirmware/Submodules/DfuDevicesScanner/Submodules/DfuFlash/View/DfuFlashViewInput.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+enum DfuFlashState: String, CaseIterable {
+    case packageSelection = "Package Selection"
+    case readyForUpload = "Ready For Upload"
+    case uploading = "Uploading"
+    case completed = "Completed"
```

Comment:
nit: what's the purpose of setting names different from cases? Is it used as a localisation key? A bit confusing.